doc/nrf/ug_thingy53.rst

```diff
@@ -285,3 +262,88 @@ To build and program the source code from the command line, complete the followi
+In that case you can flash the Thingy:53 in a similar way as nRF53 DK, see :ref:`ug_nrf5340` for details.
+Keep in mind that you will need to flash an sample or application compatible with Thingy:53 application guide to bring back serial recovery and DFU OTA.
+
+.. _thingy53_app_update_mcuboot:
+
+Firmware update using MCUboot bootloader
+----------------------------------------
+
+The Thingy:53-compatible samples and applications use MCUboot bootloader than can be used to update firmware.
+This method uses signed binary files :file:`app_update.bin` and :file:`net_core_app_update.bin` (or :file:`dfu_application.zip`).
+You can program the precompiled firmware image using one of the following ways:
+
+* Use the :doc:`mcuboot:index` feature and the built-in serial recovery mode of Thingy:53.
+  In this scenario Thingy is connected directly to your PC through USB.
+* Update the firmware over-the-air (OTA) using Bluetooth Low Energy and the nRF Programmer mobile application for Android or iOS.
+  To use this method, the application that is currently flashed on Thingy:53 must support it.
+  All precompiled images support OTA using Bluetooth.
+
+See `Updating Nordic Thingy:53 firmware`_ for the detailed procedures on how to program a Thingy:53 using nRF Connect Programmer or the nRF Programmer for Android or iOS.
+
+.. _thingy53_app_guide:
+
+Thingy:53 application guide
+***************************
+
+Thingy:53 does not have a built-in J-Link debug IC.
+Because of that, the samples and applications that are compatible with Thingy:53 by default use MCUboot bootloader with serial recovery support.
+Enabling DFU and MCUboot requires consistency in configuration of the samples and applications.
+
+For a  sample or application to be compatible with Thingy:53, the application must comply with the following configuration:
+
+* Partition manager configuration
+  Thingy:53 compatible samples use :ref:`partition_manager` to define memory partitions.
+  To ensure that partition layout does not change between builds, a sample must use static partition layout that is the same as the one in Thingy:53-compatible samples in NCS.
+  The memory layout must stay consistent, so that MCUboot can perform proper image updates and clean up the settings storage partitions.
+
+  The PCD SRAM partition is locked by the MCUboot bootloader to prevent the application from modifying the network core firmware.
+  Trying to access data on this partition results in an ARM fault.
+* MCUBoot bootloader
+  The sample must enable MCUboot bootloader and use the same configuration of the bootloader as all of the Thingy:53-compatible samples in |NCS|.
+  This is needed to let MCUBoot use the sample as an image update.
+
+In addition to the configurations mentioned above, all of the Thingy:53 compatible samples in |NCS|:
```

Comment:
Not sure how to read this and the following bullets because of the line 336. Is this something I have to make sure to implement? If so, we need `... compatible samples in the |NCS| must meet the following requirements:` or similar.
But if this is not necessary, why would I have to read the examples of implementation, as required in line 336?

Reply:
This is information about convention for Thingy:53-compatible samples in NCS. All of them provide logs over USB CDC ACM and enable DFU over BLE SMP (for convenience of end user). 
It's not necessary to enable these features to boot application with Thingy's preprogrammed MCUboot bootloader, so the application without these features could still be used as an update image. Using such application, user will not be able to DFU over BLE SMP to other application while current application is running (MCUboot's serial recovery will have to be used). The logs would not be accessible without additional external hardware too.

Reply:
OK, then maybe list the samples and applications before this part? Also, maybe rephrase to:
`... all of the Thingy:53 compatible samples in the |NCS| use the following conventions:`
And then:
`All samples use ...`
`All samples enable ...`
And I would mention the information about the possibility to disable these features, though I don't understand now if they are enabled by default or not, because you write "it is not necessary to enable these features" :/

Reply:
I would prefer to have list after, so the complete description of Thingy:53-compatible sample would be in one place